### PR TITLE
Update locale.properties "pt-br"

### PR DESCRIPTION
**Pull request by almircanella**
completed: pt-br translation.

### CODE REVIEW

**Comment by Psym3trics** [comment]
Hello, I can make the French translation if needed.

File: src/main/resources/locale.properties [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-logsEnteredAsMsg1=You have been entered as: 
-logsEnteredAsMsg2=You should be root or have configured 'udev' rules for this user to avoid any issues.
-logsFilesToUploadTitle=Files to upload:
-logsGreetingsMessage=Welcome to NS-USBloader
-logsNoFolderFileSelected=No files selected: nothing to upload.
-windowBodyConfirmExit=Data transfer is in progress and closing this application will interrupt it.\nIt's the worse thing you can do now.\nInterrupt proccess and exit?
-windowTitleConfirmExit=No, don't do this!
-btnStop=Interrupt
+btnFileOpen=Selecionar arquivos .NSP
+btnUpload=Subir para o Nintendo Switch
```
Comment (Psym3trics) [inline src/main/resources/locale.properties]: That's not a good translation at all!!!
Should we 
"Enviar para a Nintendo switch"

**Comment by Psym3trics** [comment]
> completed: pt-br translation.

Você chama isto uma tradução? Lamento imenso mas acho que até o Google tinha traduzido melhor....

**Comment by developersu** [comment]
Hey Almir!
Thank you for your work. Unfortunately I don't have enough time right last few weeks, so I'll check your PR some day next week. Just want to update you. Thank you for your patience!

**Comment by developersu** [comment]
Hi @Psym3trics, 
Yeah, there are few strings in French translated by me, but the most part of the work made by native-speaker. 
If you think I made mistakes, go ahead and let me know by sending pull request (or just leave a message, [for example, in here](https://gbatemp.net/threads/ns-usbloader-another-one-tinfoil-and-goldlead-usb-installer.533394/)).
Thanks =)

**Comment by almircanella** [comment]
Updated!

https://pastebin.com/EdSYRBSw

**Pull request closed without merging**